front-end/src/components/Dashboard/index.js

```diff
@@ -74,47 +74,61 @@ class Dashboard extends React.Component {
   }
 
   renderDashboardContent = () => {
+    return <>
```

Comment:
I think we should also have a placeholder image or their profile picture if their github info is attached

Reply:
fixed. I'm using this which says it's free. I'm not sure the best practice for citing free images, do we need to? Here is the link: https://www.iconfinder.com/icons/2931147/account_avatar_face_head_human_profile_user_icon